routes/index.js

```diff
@@ -255,6 +255,43 @@ module.exports = function () {
 
   // ===== /end docs
 
+  // Awesome
+  router.get(['/awesome', '/awesome/*'], (req, res) => {
```

Comment:
Rather than * in the path request I'd just allow for one segment i.e. just `/awesome` `/awesome/page`